cmd/gitops/beta/run/cmd.go

```diff
@@ -1045,6 +1045,7 @@ func runCommandInnerProcess(cmd *cobra.Command, args []string) error {
 							podErr error
 						)
 
+						//nolint:staticcheck
```

Comment:
Can you please add a comment why did you add this?
- https://github.com/weaveworks-liquidmetal/flintlock/blob/a19f96e2496757f09076e948c308bc2393bd245f/test/e2e/utils/runner.go#L208
- https://github.com/weaveworks-liquidmetal/flintlock/blob/a19f96e2496757f09076e948c308bc2393bd245f/infrastructure/grpc/server.go#L77

Reply:
```suggestion
						//nolint:staticcheck // deprecated, tracking issue: https://github.com/weaveworks/weave-gitops/issues/3812
```